packages/react/src/themes/teams/components/Alert/alertStyles.ts

```diff
@@ -72,9 +72,13 @@ const alertStyles: ComponentSlotStylesInput<AlertProps, AlertVariables> = {
 
     ...((p.attached === 'top' || p.attached === true) && {
       borderRadius: `${v.borderRadius} ${v.borderRadius} 0 0`,
+      marginBottom: '-1px',
```

Comment:
Would introduce the `borderWidth` with variable and use here `-${v.borderWidth}`

Reply:
@mnajdova good that you brought this up, that's exactly what I wanted to do from the beginning, but when I set `borderWidth` to `pxToRem(1)` the margin was not applied because of the inexact calculation. I can still do this but the value will be `1px` for `borderWidth` variable.

Are we ok with that?

Reply:
fixed with better solution as discussed with @kuzhelov